site/package.json

```diff
@@ -45,6 +45,7 @@
     "@xstate/inspect": "0.6.5",
     "@xstate/react": "3.0.1",
     "axios": "0.26.1",
+    "canvas": "^2.11.0",
```

Comment:
We're getting changes here because I ran into [this issue](https://stackoverflow.com/questions/14771781/nodejs-cannot-find-module-build-release-canvas) and had to re-add canvas.